lib/pattern.dart

```diff
@@ -109,7 +113,7 @@ class Glob implements Pattern {
   int get hashCode => pattern.hashCode;
 
   @override
-  bool operator ==(other) => other is Glob && pattern == other.pattern;
+  bool operator ==(Object other) => other is Glob && pattern == other.pattern;
```

Comment:
Should this be `Object?`?

Reply:
Nope-- in Dart (even as far back as 1.0) the spec says null checking is done by the runtime, so it'll never be null.